setup.cfg

```diff
@@ -44,7 +44,7 @@ install_requires =
     distlib>=0.3.1,<1
     filelock>=3.0.0,<4
     six>=1.9.0,<2   # keep it >=1.9.0 as it may cause problems on LTS platforms
-    importlib-metadata>=0.12;python_version<"3.8"
+    importlib-metadata>=3.6;python_version<"3.10"
```

Comment:
I'd like to use importlib-metadata from stdlib if available, rather than make it a dependency 🤔

Reply:
For sure. The latest commit relies once again on the old dependency. Once Python 3.6+ can be required, I'd recommend to move to this new dependency and drop `backports.entry_points_selectable`.